extensions/azurecore/src/azureResource/providers/database/databaseTreeDataProvider.ts

```diff
@@ -42,19 +42,19 @@ export class AzureResourceDatabaseTreeDataProvider extends ResourceTreeDataProvi
 				databaseName: database.name,
 				userName: database.loginName,
 				password: '',
-				authenticationType: '',
+				authenticationType: connection.AuthenticationType.AzureMFA,
```

Comment:
This was changed to be empty in https://github.com/microsoft/azuredatastudio/pull/16999 to support setting a default auth type FYI. Was there some specific issue you were seeing with that?

(one improvement here would likely be to actually save the value for the nodes - because right now if I have a default of AzureMFA but then have a couple DBs that are basic login only I don't think it's going to save the auth type correctly for those and instead always default to MFA)

Reply:
These providers are for Azure tree. Also this PR doesn't modify all providers but only for SQL Server and SQL Database that support Azure MFA. We also do that for Log Analytics and other providers that support Azure MFA, so I think this is the right thing to do.

Also, this is the only change that makes the difference, it helps select the right Azure tenant in the Connection Dialog.

Reply:
Azure servers can have basic logins though - we shouldn't assume that AzureMFA is the only auth type.

I agree that this is correct to do for AzureMFA but we should also be making sure that we handle server/database connections with the other auth types as well.

Reply:
I don't disagree but given the number of complaints regarding wrong tenant being pre-populated, that holds more priority from my perspective. We can always change it back if there's any report of concern, but I don't think that should happen since this piece of code applies to Azure tree only.

Intuitively, clicking on Connect icon in Azure Tree should be capable of defining connection properties required to instantly open a connection, as user is already authenticated, and connection details we pre-populate should be accurate since we have all the information to seamlessly login.